features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx

```diff
@@ -619,7 +621,12 @@ function TryOutController(props) {
                                             />
                                         )}
                                     >
-                                        <HelpOutline className={classes.tooltip} />
+                                        <IconButton
+                                            aria-label='Use existing Access Token or generate a new Test Key'
+                                            style={{ margin: 8, height: '48px' }}
```

Comment:
We could wrap this in a <Box /> applying these margin and height styles

Reply:
Fixed